base/shared/src/main/scala/scalaz/core/eq.scala

```diff
@@ -1,14 +1,17 @@
 package scalaz
 package core
 
+import scala.AnyRef
 import scala.language.experimental.macros
 
+@specialized
```

Comment:
so, this does absolutely nothing at all, it seems.

```
hhoughton:scalaz hhoughton$ ls base/jvm/target/scala-2.12/classes/scalaz/core/ | grep Eq
EqAnyRef.class
EqClass.class
EqSyntax$ToEqOps.class
EqSyntax.class
```

I think class-level specialization is per type parameter; you'd need to do `trait EqClass[@sp A]`.

Reply:
OK, I was never really after the `specialized`, the main idea here was the default `eq` for `AnyRef` comparisons. Let me take out the `specialized` annotations altogether.